README.md

```diff
@@ -3,14 +3,22 @@
 
 # Spectator-cpp
 
-> :warning: Experimental
+> :warning: Experimental, Telemetry Team Only
 
-Simple library for instructing code to record dimensional time series.
+This project implements a basic [Spectator] library for instrumenting C++ applications and sending
+metrics to an Atlas Aggregator service.
 
-## Description
+**<span style="color:red">This library should only be used by the Telemetry Team</span>** as a
```

Comment:
I don't think GitHub will honor the custom color.

Reply:
Hm, did not know that - verified in a test repo. Will remove.